src/nyx/nested_feature_aggregation.h

```diff
@@ -80,7 +80,7 @@ class ChildFeatureAggregation
 	}
 
 private:
-	Nyxus::AvailableChildFeatureAggregations method = Nyxus::AvailableChildFeatureAggregations::aNONE;
+	Nyxus::AvailableChildFeatureAggregations method;// = Nyxus::AvailableChildFeatureAggregations::aNONE;

```

Comment:
no default needed anymore? If yes, can we remove dead/commented code?

Reply:
Good find. Yes, inactive code needs to be cleaned away